Yank/yank.py

```diff
@@ -1413,7 +1415,20 @@ def _minimize_sampler_state(sampler_state_id, sampler_states, thermodynamic_stat
             sampler_state_id + 1, len(sampler_states), initial_energy))
 
         # Minimize energy.
-        openmm.LocalEnergyMinimizer.minimize(context, tolerance, max_iterations)
+        if (max_iterations is None) or (max_iterations == 0):
+            fire_iterations = 1000
```

Comment:
Same here.

Reply:
Ah! I see. Most of the functions I modified didn't have complete docstrings, and I hadn't even realized the function call hierarchy led back up to `AlchemicalPhase.minimize()`, so I had no idea what the behavior was supposed to be. Having docstrings even for private methods will make it easier in future to ensure we keep the contract the function makes with its caller.

I'll add these in and modify the behavior to match the original contract.